hyperswitchSDK/Core/PaymentMethodManagement/PaymentMethodMangement.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  PaymentMethodMangement.swift
+//  hyperswitch
+//
+//  Created by Shivam Nan on 15/09/24.
+//
+
+import Foundation
+import WebKit
+
+public class PaymentMethodMangement {
+    @Published var ephemeralKey: String?
+    
+    init(ephemeralKey: String){
+        self.ephemeralKey = ephemeralKey
+    }
+    
+    /// Method to present the payment sheet view with a given root view and completion handler.
+    private func presentWithRootView(from presentingViewController: UIViewController, rootView: RCTRootView) {
+        let viewController = UIViewController()
```

Comment:
Please use PaymentSheetUIViewController instead of this, you can rename PaymentSheetUIViewController to HyperUIViewController.

Reply:
Done, as per suggested. 👍